### PR TITLE
FareZone and refs should have constraint checks in timetable XSD

### PR DESCRIPTION
**Pull request by syversenkr**
As you are allowed to define tariffZones->FareZone in both SiteFrame and ServiceFrame

### CODE REVIEW

File: xsd/NeTEx_publication_timetable.xsd [hunk 1]
```diff
@@ -1020,6 +1020,27 @@ Provides a general purose wrapper for NeTEx data content.</xsd:documentation>
...
+			<xsd:annotation>
+				<xsd:documentation>Every [FareZone Id + Version] must be unique within document.</xsd:documentation>
+			</xsd:annotation>
+			<xsd:selector xpath=".//netex:FareZone"/>
+			<xsd:field xpath="@id"/>
+			<xsd:field xpath="@version"/>
+		</xsd:unique>
+		<!-- =====FareZone Key ========================== -->
+		<xsd:keyref name="FareZone_KeyRef" refer="netex:FareZone_AnyVersionedKey">
+			<xsd:selector xpath=".//netex:FareZoneRef | .//netex:ParentFareZoneRef | .//netex:FromFareZoneRef | .//netex:ToFareZoneRef   |.//netex:ProjectedObjectRef"/>
```
Comment (skinkie) [inline xsd/NeTEx_publication_timetable.xsd:1035]: Double space after "ToFareZoneRef"

**Comment by nick-knowles** [comment]
Was added to new modes branch some time back

**Comment by skinkie** [comment]
@nick-knowles please refer to the commit.

File: xsd/NeTEx_publication_timetable.xsd (hunk 1 above)
Comment (ue71603) [reply xsd/NeTEx_publication_timetable.xsd:1035]: @syversenkr : Please fix the double space.

Comment (skinkie) [inline xsd/NeTEx_publication_timetable.xsd:1035]:
```suggestion
			<xsd:selector xpath=".//netex:FareZoneRef | .//netex:ParentFareZoneRef | .//netex:FromFareZoneRef | .//netex:ToFareZoneRef | .//netex:ProjectedObjectRef"/>
```

Comment (skinkie) [reply xsd/NeTEx_publication_timetable.xsd:1035]: @syversenkr can you accept the suggestion?

**Comment by skinkie** [comment]
> Was added to new modes branch some time back

They are in NeTEx_publication.xsd but not in NeTEx_publication_timetable.xsd, this pull request is about the latter.

**Review by skinkie — APPROVED** [review]

**Review by ue71603 — APPROVED** [review]

**Review by Aurige — APPROVED** [review]

**Pull request merged**